### DISCUSSION TITLE
React Native

### DISCUSSION THREAD

**Discussion by ColeTownsend** [shuding/react-wrap-balancer#11] [Ideas]
I don't think anything like this exists for React Native! That would really separate this library from others out there.

**Comment by shuding** [comment]
For RN support, we will need native APIs as alternative to `ResizeObserver` and `.offsetWidth` (or `getBoundingRect`). Maybe it's doable?

**Reply by wobsoriano** [reply]
Not sure if there's any `ResizeObserver` API for RN but I think the [LayoutEvent](https://reactnative.dev/docs/layoutevent) of the parent `View` could suffice.

```js
{
    layout: {
        width: 520,
        height: 70.5,
        x: 0,
        y: 42.5
    },
    target: 1127
}
```
